test/unit/argument_parser/format_parse_test.cpp

```diff
@@ -969,3 +969,16 @@ TEST(parse_test, issue1544)
         EXPECT_EQ(foobar_option_value, "ballo");
     }
 }
+
+TEST(parse_test, is_option_set)
+{
+    const char * argv[] = {"./argument_parser_test", "-l", "hallo", "--foobar", "ballo", "--", "--loo"};
+    seqan3::argument_parser parser{"test_parser", 5, argv, false};
+
+    EXPECT_TRUE(parser.is_option_set('l'));
+    EXPECT_TRUE(parser.is_option_set("foobar"));
```

Comment:
Related to my comments for `is_option_set`, I think there are a few cases missing:

1. What if I add another short option that is the last option, or in between?
2. Same for long options.

Reply:
I don't exactly know what you mean.. can you give a command line example?

Reply:
It's ok now after explaining the `--` thing